src/AppInitializer.tsx

```diff
@@ -62,9 +61,16 @@ class AppInitializer extends React.Component<{},AppInitializerToken> {
 
   render() {
     return (
-      <TokenContext.Provider value={this.state.token}>
+      <SynapseContextProvider
```

Comment:
Add access token to the SynapseContextProvider.  Again, overkill for this simple application where it only has a single child (but useful for portals).